adb_graphics/figures/maps.py

```diff
@@ -375,7 +382,7 @@ def _draw_field(self, ax, field, func, **kwargs):
         '''
 
         x, y = self._xy_mesh(field)
-        vals = field.values()[::]
+        vals = field.values()
```

Comment:
It appears to be unnecessary.  I think I removed it at one time to try and locate the source of an error, and though it wasn't the source, I left the change because it didn't seem to cause an error or have an effect.  I've tested it this way with all the models, and the result is the same with or without it.